docs/platform/visual/whats-new-beta-3.md

```diff
@@ -0,0 +1,427 @@
+    });
+    
+    // New syntax
+    BaseModel.extend({
+     $type: {
+       props: [
+         {
+           name: "measure",
+           base: "pentaho/visual/role/property",
+           modes: [
+             {dataType: "number", isContinuous: false}
+           ]
+         }
+       ]
+     }
+    });
+    ```
+
+7. Quantitative visual role which accepts zero or more number fields
+
+    ```js
+    // Old syntax
+    BaseModel.extend({
+     $type: {
+       props: [
+         {
+           name: "measure",
+           base: "pentaho/visual/role/property",
+           levels: ["quantitative"],
+           dataType: "number"
+         }
+       ]
+     }
+    });
+    
+    // New syntax
+    BaseModel.extend({
+     $type: {
+       props: [
+         {
+           name: "measure",
+           base: "pentaho/visual/role/property",
+           modes: [{dataType: ["number"]}]
+         }
+       ]
+     }
+    });
+    ```
+    
+    Note that the data type `["number"]` denotes that the "composite value" of the visual role is a list of numbers.
+
+### Renamed Mapping `attributes` to `fields`
+
+The `View` code needs to know which fields are mapped to a visual role.
+The visual role `Mapping` class was changed to expose the list of mapped fields under the 
+[pentaho.visual.role.Mapping#fields]({{site.refDocsUrlPattern | replace: '$', 'pentaho.visual.role.Mapping#fields'}})
+property.
+
+Example of old and new view code to obtain the column indexes of the fields mapped to the visual roles:
+
+```js
+// Old code
+var model = this.model;
+
+// The name of the data attributes that are mapped to the visual roles
+var categoryAttribute = model.category.attributes.at(0).name;
+var measureAttribute = model.measure.attributes.at(0).name;
+
+// Their column indexes in the data table.
+var categoryColumn = dataTable.getColumnIndexByAttribute(categoryAttribute);
+var measureColumn = dataTable.getColumnIndexByAttribute(measureAttribute);
+
+          
+// New code
+var model = this.model;
+
+// The name of the data attributes that are mapped to the visual roles
+var categoryFieldName = model.category.fields.at(0).name;
+var measureFieldName = model.measure.fields.at(0).name;
+
+// Their column indexes in the data table.
+var categoryFieldIndex = dataTable.getColumnIndexById(categoryFieldName);
+var measureFieldIndex = dataTable.getColumnIndexById(measureFieldName);
+```
+
+Note that, additionally, the data table method `getColumnIndexByAttribute` has been renamed to `getColumnIndexById`.
+
+### New _Scenes_ helper class to speed up writing the code of Views{{site.starNew}}
+
+The code of a `View` class is, in part, 
+made up of code that transforms the data in the data table into a form which is suitable for rendering.
+This form is typically written in the namespace of visual roles, such as in: 
+the value of `x` is `1` and the value of `series` is `Portugal`. 
+The fields that were mapped to visual roles are not relevant for most rendering purposes.
```

Comment:
Not very clear. You are referring to field **names**, right? Because the field **values** are, of course, relevant.